examples/oidc/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+
+services:
+
+  james:
+    image: apache/james:memory-3.6.1
```

Comment:
Won't support the X-User stuff. Please use `latest` or `3.7.0` to make it clear that the latest master is required for this to work.

Reply:
Good remark